Author: Harald Sitter
Description: give more time for akonadi-server to start
 10 seconds is way too short timeframe. To make things worse, kdepimlibs just
 calls exit() after showing error detection dialog. The patch was stolen from
 Kubuntu.
Origin: vendor, http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/lucid/kdepimlibs/lucid/download/head:/kubuntu_01_increase_-20100413181428-vr64cd6ivtkqbzhp-2/kubuntu_01_increase_fail_timeout.patch

--- a/akonadi/control.cpp
+++ b/akonadi/control.cpp
@@ -133,7 +133,7 @@
   kDebug() << "Starting Akonadi (using an event loop).";
   mEventLoop = new QEventLoop( mParent );
   // safety timeout
-  QTimer::singleShot( 10000, mEventLoop, SLOT(quit()) );
+  QTimer::singleShot( 60000, mEventLoop, SLOT(quit()) );
   mEventLoop->exec();
   mEventLoop->deleteLater();
   mEventLoop = 0;
